recipes/alembic/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conans import ConanFile, CMake, tools
+import os
+import shutil
+
+
+class AlembicConan(ConanFile):
+    name = "alembic"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/alembic/alembic"
+    description = "Open framework for storing and sharing scene data."
+    topics = ("usb", "scene")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_hdf5": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_hdf5": False
+    }
+    generators = "cmake_find_package"
+    exports_sources = "AlembicIlmBase.cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("openexr/2.5.7")
+        if self.options.with_hdf5:
+            self.requires("hdf5/1.12.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["USE_ARNOLD"] = False
+        self._cmake.definitions["USE_MAYA"] = False
+        self._cmake.definitions["USE_PRMAN"] = False
+        self._cmake.definitions["USE_PYALEMBIC"] = False
+        self._cmake.definitions["USE_BINARIES"] = False
+        self._cmake.definitions["USE_EXAMPLES"] = False
+        self._cmake.definitions["USE_HDF5"] = self.options.with_hdf5
+        self._cmake.definitions["USE_TESTS"] = False
+        self._cmake.definitions["ALEMBIC_BUILD_LIBS"] = True
+        self._cmake.definitions["ALEMBIC_ILMBASE_LINK_STATIC"] = True # for -DOPENEXR_DLL, handled by OpenEXR package
+        self._cmake.definitions["ALEMBIC_SHARED_LIBS"] = self.options.shared
+        self._cmake.definitions["CMAKE_CXX_STANDARD"] = 11
+        self._cmake.configure(source_folder=self._source_subfolder, build_folder=self._build_subfolder)
```

Comment:
You really need to add a cmake wrapper that includes `conanbuildinfo.cmake` + calls `conan_basic_setup()`.
Until conan 2.0, this helpers sets some critical variables (such as switching between libstdc++ and libstdc++11)

Reply:
Yeah, looks like I need to do that. Thanks.